composer.json

```diff
@@ -11,17 +11,17 @@
     "require": {
         "php": ">=5.3.3",
         "giggsey/libphonenumber-for-php": "~5.7|~6.0|~7.0|~8.0",
-        "symfony/framework-bundle": "~2.1|~3.0"
+        "symfony/framework-bundle": "~2.1|~3.0|^4.0"
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "~1.0",
-        "jms/serializer-bundle": "~0.11|~1.0",
+        "jms/serializer-bundle": "~0.11|~1.0|~2.0",
         "phpunit/phpunit": "~4.0",
```

Comment:
Looks like phpunit 4.x doesn't really like Symfony 4: https://travis-ci.org/misd-service-development/phone-number-bundle/jobs/303934028#L761. You can try adding `|^5.7` here to get the build working.